ml-conversational-analytic-tool/utils.py

```diff
@@ -21,14 +21,19 @@ def export_to_cvs(export_df: pd.DataFrame, name):
     print("Output file: ", os.path.abspath(file))
 
 
-def construct_file_name(name, raw_datafile, suffix):
+def construct_file_name(name, component_a, component_b, separator="_"):
```

Comment:
I think it's always better to be specific with the names. To me, `component_a` and `component_b` don't specify what needs to be passed into the function. Can we keep the original or choose better names than components?

Reply:
The original names do not fit anymore in the extended use of this function. 
I still do not have an idea for better names...

Reply:
Based on how it's used `utils.construct_filename(args.name, args.organization, args.reponame)` and the help description `... If not specified, the name is constructed like this: <organization>_<reponame>.csv`

I think appropriate names could be organization and reponame? WDYT?

Reply:
It's used a little differently in featureVector.py

Reply:
Got it, thanks for clarifying. I do see that it's being used differently in the featureVector.py and github_data.py.

I am okay with component_a and component_b as the name.